widget_tweaks/templatetags/widget_tweaks.py

```diff
@@ -45,7 +47,6 @@ def set_attr(field, attr):
 
     def process(widget, attrs, attribute, value):
         attrs[attribute] = value
-
```

Comment:
Separate the whitespace changes from the functional change commits. Make a separate commit and PR for altering separate things, please.